chart/templates/_helpers.yaml

```diff
@@ -653,6 +664,14 @@ Create the name of the cleanup service account to use
     airflow jobs check --job-type TriggererJob --hostname $(hostname)
 {{- end }}
 
+{{define "dag_processor_liveness_check_command"}}
+  - sh
+  - -c
+  - |
+    CONNECTION_CHECK_MAX_COUNT=0 AIRFLOW__LOGGING__LOGGING_LEVEL=ERROR exec /entrypoint \
+    airflow jobs check --job-type SchedulerJob --hostname $(hostname)
```

Comment:
We don't have a dedicated jobs check command for standalone dag processor for now. I can create in this same PR or in a separate one.

Reply:
In the meantime, you can probably leave off the job type and it'll still work.

Reply:
Makes sense